tmail-backend/jmap/extensions-es-v7/src/main/java/com/linagora/tmail/james/jmap/module/ContactIndexCreationUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.linagora.tmail.james.jmap.module;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import org.apache.james.backends.es.v7.ElasticSearchConfiguration;
+import org.apache.james.backends.es.v7.IndexCreationFactory;
+import org.apache.james.backends.es.v7.ReactorElasticSearchClient;
+
+import com.linagora.tmail.james.jmap.ContactMappingFactory;
+import com.linagora.tmail.james.jmap.ElasticSearchContactConfiguration;
+
+public class ContactIndexCreationUtil {
+    public static ReactorElasticSearchClient createIndices(ReactorElasticSearchClient client,
```

Comment:
void is better?
We create two indexes, but we only return 1 result in them. 
IMO it will have a bit confuse for the first read